### ISSUE TITLE
The prepdocs remove flag/instructions are confusing

### ISSUE THREAD

**Issue by notanaha** [Azure-Samples/azure-search-openai-demo#1503]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ x ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
As described in [Removing documents](https://github.com/Azure-Samples/azure-search-openai-demo/blob/main/docs/data_ingestion.md#removing-documents) section, added --remove flag in scripts/prepdocs.ps1, but all the documents were removed from the index, but those in blob storage were left.

### Any log messages given by the failure
>

### Expected/desired behavior
>

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### azd version?
> run `azd version` and copy paste here.
<img width="586" alt="img2" src="https://github.com/Azure-Samples/azure-search-openai-demo/assets/31022537/11b2d78e-e900-4662-a6e0-9034275e80c5">

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by pamelafox** [comment]
You're saying that it did remove the blobs from blob storage, but did not remove them from the search index?

**Comment by notanaha** [comment]
No, opposite meaning, i.e., index was cleared, but blog wasn't.
The more serious issue would be that while I used --remove flag, all data was removed  from the index. 
Please not I didn't used --removeall flag.

**Comment by pamelafox** [comment]
Did you modify the dataArg as well? I can't see from your screenshot.

**Comment by notanaha** [comment]
No, I just followed the instruction. If my modification was incorrect, please add such instruction.

![image](https://github.com/Azure-Samples/azure-search-openai-demo/assets/31022537/1399f036-7d5a-4f3b-a528-e6921d1dc38e)

**Comment by pamelafox** [comment]
The second part of the instruction is to replace "/data/*" with "/data/your-filename.pdf", so you need to do that as well, otherwise it operates on all of "/data/*".

I think we need to make it possible to just specify --remove and --removeall from the command line, so that you don't have to mess with that file, as it is fairly confusing to wade through all that. We could also make it so that "remove" would only operate on a single file, but that would mean you couldn't use it with globs.

**Comment by notanaha** [comment]
So, with remove flag, did I need to move files from /data folder?
I tried it prior to open this issue, but didn't work as intended like so.
I agree that this is concerned about the explanation matter.
